frontend/src/components/Post.js

```diff
@@ -1,79 +1,67 @@
-        }
-    }
+    useEffect(() => {
+        loadPicture();
+    }, [props.id])
 
-    loadPicture() {
-        axios.get(global.config.BACKEND_URL + "/picture/" + this.props.id).then(
+    const loadPicture = () => {
+        axios.get(global.config.BACKEND_URL + "/picture/" + props.id).then(
             (response) => {
-                this.setState({ Picture: response.data });
+                setPicture(response.data);
             }
         ).catch(
             () => {
-                this.setState({
-                    Picture: null
-                });
+                setPicture(null);
             }
         )
     }
 
-    onCommentPosted() {
+    const onCommentPosted = () => {
         //Whenver a comment is posted, inverse the boolean associated to refreshComment
         //This state will be passed into a commentList in order to make the component rerender
-        this.setState({ refreshComment: !this.state.refreshComment })
+        setRefreshComment(!refreshComment);
     }
 
-    render() {
-        return (
-            <div>
-                {this.state.Picture ?
-                    <div className="post">
-                        <div className="image-wrapper">
-                            <img src={`${global.config.BACKEND_URL}/picture/${this.props.id}.jpg`}
-                                alt={`${this.props.id}`} />
-                        </div>
-                        <div className="text-wrapper">
-                            {/* a Description component can be created to facilitate the creation of Post components */}
-                            <div className="post-description">
-                                <Link to={`/account/${this.state.Picture.account}`}>
-                                    {this.state.Picture.account}
-                                </Link>: {this.state.Picture.caption}
-                            </div>
-                            <div className="comments">
-                                <CommentList refreshComment={this.state.refreshComment} postId={this.props.id} />
-                            </div>
-                            <div className="postsComment">
-                                <PostComment postId={this.props.id} onCommentPosted={this.onCommentPosted.bind(this)} />
+    return (
+        <div>
+            {Picture ?
+                <div className="post">
+                    <div className="image-wrapper">
+                        <img src={`${global.config.BACKEND_URL}/picture/${props.id}.jpg`}
+                            alt={`${props.id}`} />
+                    </div>
+                    <div className="text-wrapper">
+                        {/* a Description component can be created to facilitate the creation of Post components */}
+                        <div className="post-description">
+                            <div className="accountName">
+                                <Link to={`/account/${Picture.account}`}>
```

Comment:
Hyphenate to keep the format consistent with all other class names: account-name